pkgs/development/python-modules/magic-wormhole/default.nix

```diff
@@ -37,6 +37,10 @@ buildPythonPackage rec {
     sed -i -e "s|'ifconfig'|'${nettools}/bin/ifconfig'|" src/wormhole/ipaddrs.py
   '';
 
+  postInstall = ''
+    install -Dm644 docs/wormhole.1 $out/share/man/man1/wormhole.1
```

Comment:
Should we do symlink this to ``magic-wormhole`` to match the package name and the binary name?

Reply:
I think the executable is
```
❯ tree result/bin
result/bin
└── wormhole

0 directories, 1 file
```

so it's fine.